src/bundle/Resources/public/js/scripts/widgets/flatpickr.js

```diff
@@ -0,0 +1,8 @@
+import flatpickrLanguages
+    from '../../../../../../../../../../public/bundles/ezplatformadminuiassets/vendors/flatpickr/dist/l10n';
+
+(function (eZ, flatpickr) {

```

Comment:
```suggestion
(function (global, doc, eZ, flatpickr) {
```
these two parameters (global and doc) should be always included, even if they're not used at the moment, as for consistency and readability they should be as first two args, and we shouldn't change order of params in the future... I think. @dew326